src/openrct2/sprites.h

```diff
@@ -802,6 +802,23 @@ enum {
 
     SPR_G2_MINI_RC_BOOSTER_NE_SW = SPR_G2_BEGIN + 91,
     SPR_G2_MINI_RC_BOOSTER_NW_SE = SPR_G2_BEGIN + 92,
+        
+    SPR_G2_MINIATURE_RAILWAY_QUARTER_TURN_3_TILES_SW_SE_PART_3 = SPR_G2_BEGIN + 93,
+        
+    SPR_G2_MINIATURE_RAILWAY_BEGIN = SPR_G2_BEGIN + 94,
```

Comment:
Alignment looks a bit odd here. Same for the last entry.
Both forms of alignment are fine, but you must pick one for a block ;-)